composer.json

```
@@ -25,15 +25,18 @@
         "drupal/acquia_cms_site_studio": "dev-develop",
         "drupal/acquia_cms_starter": "dev-develop",
         "drupal/acquia_cms_toolbar": "dev-develop",
+        "drupal/acquia_cms_toolbar_gin": "^1.0",
         "drupal/acquia_cms_tour": "dev-develop",
         "drupal/consumer_image_styles": "^4.0",
+        "drupal/gin_toolbar": "^1.0@RC",
```

Instead of gin_toolbar module add gin theme as it's dependency of Gin theme.
```suggestion
        "drupal/gin": "^3.0",
```